addons/html_builder/static/src/interactions/carousel.edit.js

```diff
@@ -13,7 +12,23 @@ export class CarouselEdit extends Interaction {
             "t-on-keydown": this.onControlKeyDown,
             "t-att-class": () => ({ o_we_no_overlay: true }),
         },
+        ".carousel-control-prev, .carousel-control-next": {
+            "t-att-data-bs-slide": () => undefined,
+            "t-on-mousedown": this.onMouseDown,
+        },
+        ".carousel-indicators > *": {
+            "t-att-data-bs-slide-to": () => undefined,
+            "t-on-mousedown": this.onMouseDown,
+        },
     };
+
+    /**
+     * Redirects a carousel control click on the active slide.
+     */
+    onMouseDown() {
+        this.el.querySelector(".carousel-item.active").click();
+    }
+
```

Comment:
You can remove the `t-on-mousedown` I think, and therefore remove the `onMouseDown` too => it is not needed anymore since `onControlClick` and `onControlKeydown` work :slightly_smiling_face: 
We just need the fact that `data-bs-*` attributes are removed :slightly_smiling_face: